ihmc-perception/src/main/java/us/ihmc/perception/realsense/RealsenseDevice.java

```diff
@@ -83,6 +84,9 @@ public RealsenseDevice(rs2_context context, rs2_device device, String serialNumb
       realsense2.rs2_config_enable_stream(config, realsense2.RS2_STREAM_DEPTH, DEPTH_STREAM_INDEX, depthWidth, depthHeight, realsense2.RS2_FORMAT_Z16, fps, error);
       checkError("Failed to enable stream.");
 
+      BytePointer serialNumber = realsense2.rs2_get_device_info(device, RS2_CAMERA_INFO_SERIAL_NUMBER, error);
```

Comment:
Instead of passing in the serial number (which can be null), we get it from the device

Reply:
Is it possible to not get the serial number? If its coming from the device itself what would that mean?

Reply:
I think it's useful to keep it. Puts it in the log.